contracts/WildcardSteward_v2.sol

```diff
@@ -48,6 +48,9 @@ contract WildcardSteward_v2 is Initializable {
 
     MintManager_v2 public mintManager;
 
+    mapping(uint256 => address) artistAddresses; //mapping from tokenID to the artists address
```

Comment:
Keep track of which artist made which token and have an address to pay them

Reply:
I'm a tiny bit concerned about putting too much logic into the steward, overtime it might become bloted with lots of logic; having separate, modular, independant and well defined interface contracts would make this easier (both to reason about, and to test).

We could keep this separate and have a function that handles paying out to artists (similar to how the mint manager is separate.

Not strongly opposed to to putting it directly in the steward, just raising a concern